src/symbolize/gimli/mmap_unix.rs

```diff
@@ -15,14 +15,14 @@ pub struct Mmap {
 }
 
 impl Mmap {
-    pub unsafe fn map(file: &File, len: usize) -> Option<Mmap> {
+    pub unsafe fn map(file: &File, len: usize, offset: i64) -> Option<Mmap> {
```

Comment:
hmm, maybe offset should be a u64? I'm not seeing how an negative offset is ever valid? I know it's an `off_t` in the API but like... we're not going to be passing a value greater than `i64::MAX` in.

Reply:
Yeah I agree